### PR TITLE
Fix dump database script - dump location

### PR DESCRIPTION
**Pull request by kael89**
After I recently move those scripts under the `packages/database` folder, the `dumpDatabase` script will download the dump file in that location.

The previous behaviour was downloading the dump in the monorepo root by default. This is also assumed by the `yarn refresh-database` script defined in the root `package.json`, which expects the dump to be in the root. This script re-instates the previous behaviour regarding dump default locations

### CODE REVIEW

File: packages/devops/scripts/ci/validateBranchName.sh [hunk 1]
```diff
@@ -60,7 +60,6 @@ function validate_name_chars() {
 }
 
 branch_name=$(get_branch_name)
-check_name_is_not_reserved $branch_name
```
Comment (edmofro) [inline packages/devops/scripts/ci/validateBranchName.sh:63]: Why this change?

**Review by edmofro — COMMENTED** [review]
This happened to me and I didn't realise why, just thought I must have called the script from the wrong directory! Thanks for fixing. Just the question about the branch name validation, otherwise it all looks good.

File: packages/devops/scripts/ci/validateBranchName.sh (hunk 1 above)
Comment (kael89) [reply packages/devops/scripts/ci/validateBranchName.sh:63]: Fixes an existing (non breaking issue) - the function `check_name_is_not_reserved ` was removed, but its call was not.

This function used to ensure that the branch was not `e2e`, now we maintain that branch (clone of `master`)

Comment (edmofro) [reply packages/devops/scripts/ci/validateBranchName.sh:63]: Gotcha

**Review by edmofro — APPROVED** [review]
